src/enums/affectedStatus.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+This file is part of CycloneDX JavaScript Library.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) OWASP Foundation. All Rights Reserved.
+*/
+
+export enum AffectedStatus {
```

Comment:
rename!?

XML schema calles it `impactAnalysisAffectedStatusType`
JSON schema has no name or it, uses it as property `affectedStatus`

Reply:
done via 877d012731e1c684e0ef472698cb8a39c232af3f